### PR TITLE
Tutorials

### PR DESCRIPTION
**Pull request by akhoubani**

### CODE REVIEW

File: docs/tuto_simul_exec.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+
+<!-- #endregion -->
+
+## Initialize and setup up simulation parameters (recap)
+
+
+In the previous tutorial we saw how to install the packages and setup a simulation run. Let us do it here in one step.
+
+```python
```
Comment (paugier) [inline docs/tuto_simul_exec.md:23]: You should replace ````python` by ````{code-cell}` everywhere.

File: docs/examples/snek5000-tgv/ref_data [hunk 2]
```diff
@@ -0,0 +1,2003 @@
+# Source: http://dept.ku.edu/~cfdku/hiocfd/data.tgz
```
Comment (paugier) [inline docs/examples/snek5000-tgv/ref_data:1]: Could you please change the name of this file to ref_data_spectral_code.csv ?

File: docs/tuto_simul_exec.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
+---
+jupytext:
+  text_representation:
+    format_name: myst
+kernelspec:
+  display_name: Python 3
+  name: python3
+execution:
+  timeout: 100
```
[the remark this answers is not in the document]
Comment (akhoubani) [reply docs/tuto_simul_exec.md:9]: I changed the parameters, but the simulation result is not satisfactory! I'll try to get it shorter.

[the remark this answers is not in the document]
Comment (akhoubani) [reply docs/tuto_simul_exec.md:9]: I'll try to get `Phill` case shorter.

**Comment by paugier** [comment]
@akhoubani I tried to build the doc locally. Some of the simulations are much to long! You need to make them much much shorter.

Just what is needed to show how snek5000 works. They can be very short in time and very small in space. And bad quality also.

**Comment by paugier** [comment]
Ah strange error on CI for doc:

```
2022-10-28T19:17:13.6776436Z  set initial conditions
2022-10-28T19:17:13.6777733Z  nekuic (1) for ifld            1
2022-10-28T19:17:13.6882102Z  call nekuic for vel  
2022-10-28T19:17:13.7003689Z  xyz min     0.0000       0.0000       0.0000    
2022-10-28T19:17:13.7009018Z  uvwpt min  -1.0000      -1.0000       0.0000       0.0000       0.0000    
2022-10-28T19:17:13.7012865Z  xyz max     6.2832       6.2832       6.2832    
2022-10-28T19:17:13.7013206Z  uvwpt max   1.0000       1.0000       0.0000       0.0000       0.0000    
2022-10-28T19:17:13.7013666Z  done :: set initial conditions
2022-10-28T19:17:13.7013963Z   
2022-10-28T19:17:13.7014777Z   ERROR:  Non-positive diffusivity (    0.00    ) specified for field 1.
2022-10-28T19:17:13.7015814Z ABORTING in VPROPS
```

Do you understand what it means? And why we have this? It passes for me locally.

**Pull request merged**